BackendApartmentReservation/BackendApartmentReservation/WebApi/ValidateModelStateAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using BackendApartmentReservation.Extensions.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using NLog;
+
+namespace BackendApartmentReservation.WebApi
+{
+    public class ValidateModelStateAttribute : ActionFilterAttribute
```

Comment:
What was the reason behind `Attribute` instead of `Filter`? Filters will be used on all of the controllers before request, so you wont have to put this attribute on every controller method